apps/omg_watcher/lib/omg_watcher/exit_processor.ex

```diff
@@ -273,8 +273,12 @@ defmodule OMG.Watcher.ExitProcessor do
 
   def handle_call({:finalize_in_flight_exits, finalizations}, _from, state) do
     _ = if not Enum.empty?(finalizations), do: Logger.info("Recognized ife finalizations: #{inspect(finalizations)}")
-    {:ok, state, db_updates} = Core.finalize_in_flight_exits(state, finalizations)
-    {:reply, {:ok, db_updates}, state}
+
+    {:ok, state, db_updates, input_exits, output_exits} = Core.finalize_in_flight_exits(state, finalizations)
+    {:ok, input_exits_state_updates, _} = State.exit_utxos(input_exits)
+    {:ok, output_exits_state_updates, _} = State.exit_utxos(output_exits)
```

Comment:
there still is a large step missing here, which is taking validities into account, to cover the case where an IFE finalizes invalidly. But that's another large leap to take, so perhaps another issue/PR. And it would require to introduce the concept of a late invalid IFE, so that SEs and IFEs finally behave on par.

Maybe as a bare minimum, grab the invalid utxos (second list in the `_`-out tuple in the return), concatenate both flavors (inputs outputs) and if that list is non-empty (some IFE in-out-puts finalized invalidly) print a `Logger.warn` with some message and positions (make sure to inspect).

Leaving a TODO or a NOTE and reference to an issue is also desired

Reply:
done